### ISSUE TITLE
Issues about FPS during multi tasks

### ISSUE THREAD

**Issue by zhuzhuxia1221** [dusty-nv/jetson-inference#1317]
I ran two tasks (both cityscapes-512x256) simultaneously on Jetson Xavier and then turned out that FPS doubled compared to running a single task. However, Then I increased the resolution to 2048x1024 and it resulted in FPS cutting down a half when I launched another task. I wonder why it happened. The screenshots are attached to this comment.

![photo1638871707](https://user-images.githubusercontent.com/92303648/145009579-9bebb915-7e22-4211-98c6-d16a81d08f3b.jpeg)

![photo1638871707 (1)](https://user-images.githubusercontent.com/92303648/145009751-fed62829-d77d-471b-ae5a-292cc12847c0.jpeg)

![photo1638871707 (2)](https://user-images.githubusercontent.com/92303648/145009833-4da079e2-47ff-451f-b933-eea4a807c142.jpeg)

![photo1638871707 (3)](https://user-images.githubusercontent.com/92303648/145009913-5e51eaa9-0a42-4768-99c4-e59e49e3f9b0.jpeg)

Looking forward to u reply.

**Comment by dusty-nv** [comment]
I think it's because the more complex 2048x1024 model uses 100% of GPU, and hence it reduces framerate when you run multiple.

You can keep an eye on the GPU utilization using the `tegrastats` utility.

**Comment by zhuzhuxia1221** [comment]
Thank you for your reply.
Did you mean that the function of net.GetNetworkFPS() actually means the sum of the frames of all the running tasks?

**Comment by dusty-nv** [comment]
`net.GetNetworkFPS()` returns how long it took to process that particular model with TensorRT.   However if you have other processes running using the GPU, the GPU may do time-sharing between them, so it can slow down a model.   

If you wrote a single application/script that ran the same model twice on different streams, then I believe you should see normal performance reported from `net.GetNetworkFPS()` because the models would be executed in serial as opposed to in parallel.
